### ISSUE TITLE
[Category] falliing nether_portal block crashes game upon existence

### ISSUE THREAD

**Issue by ReiaTheGeia** [sp614x/optifine#7082]
## Description of Issue
running any command to summon a falling_block nether_portal such as `/summon minecraft:falling_block ^ ^3 ^3 {BlockState:{Name:"minecraft:nether_portal"}}` crashes the game after the world automatically closes out.

Unknown if the world will be permanently lost but when I first encountered this the world wouldn't open and would immediately crash, although this might have been before the full release of 1.19.2-Optifine_HD_H9

## Steps to Reproduce
1. Boot up a test world that you don't mind losing
2. run the following command: `/summon minecraft:falling_block ^ ^3 ^3 {BlockState:{Name:"minecraft:nether_portal"}}`
3. Wait for the world to save and the game to close itself
4. Be met with the following:
![image](https://user-images.githubusercontent.com/58919834/187048756-878d49f9-769f-46fb-95f1-7b60b077356f.png)

## OptiFine Version
1.19.2-Optifine_HD_H9

## Installation Method
Installer

## Log Files/Crash Reports
[crash-2022-08-27_15.27.11-client.txt](https://github.com/sp614x/optifine/files/9438713/crash-2022-08-27_15.27.11-client.txt)
[crash-2022-08-27_15.19.32-client.txt](https://github.com/sp614x/optifine/files/9438714/crash-2022-08-27_15.19.32-client.txt)

## F3 Debug Screenshot
![2022-08-27_15 37 49](https://user-images.githubusercontent.com/58919834/187048926-63f26afb-1f4d-4154-b00a-038aad9b7f53.png)

## Prior Testing
- Does not happen in Vanilla.

## Additional Information
- Nothing I can think of

**Comment by ReiaTheGeia** [comment]
Persists in 1.19.2-Optifine_HD_I1_pre1, in which it crashes the launcher, also. (the same error message pops up upon re-opening the launcher)
ibid for 1.19.2-Optifine_HD_I1_pre2

**Comment by vinc3nz** [comment]
Does this happen in vanilla too?

**Comment by ReiaTheGeia** [comment]
No, if you read the original bug report it does not happen in vanilla.

**Comment by ReiaTheGeia** [comment]
I'll probably test this again after Optifine 1.19.3 releases

**Comment by ReiaTheGeia** [comment]
Finally got to testing this again and persists to Optifine Release 1.19.4 I3
